src/MongoDB.Driver.Core/Core/NativeLibraryLoader/INativeLibraryLoader.cs

```diff
@@ -0,0 +1,24 @@
+﻿/* Copyright 2019-present MongoDB Inc.
+*
+* Licensed under the Apache License, Version 2.0 (the "License");
+* you may not use this file except in compliance with the License.
+* You may obtain a copy of the License at
+*
+* http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an "AS IS" BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+*/
+
+using System;
+
+namespace MongoDB.Driver.Core.NativeLibraryLoader
+{
+    internal interface INativeLibraryLoader
+    {
+        IntPtr GetFunction(string name);
```

Comment:
Let's call this method `GetFunctionPointer` .

Having two methods in different places called `GetFunction` is confusing.